docs/plugindev/README.md

```diff
@@ -259,6 +259,40 @@ Note: as the configuration variable implies, the
 plugin development and testing only. The use of this setting in the production setting
 is not supported.
 
+### Testing the plugins
+
+Plugin tests can be run by installing the admin `test` plugin.
+Currently, we only support testing plugins build on local.
+
```

Comment:
How about adding a note which explains the plugin is deprecated and we are not planning to enhance this in the future? cc @marckhouzam @vuil @mpanchajanya 

```suggestion

The `test` admin functionality has been deprecated and no future enhancements are planned for this plugin.

```

Reply:
I think that is a good idea.